app/src/main/java/com/kms/cura/view/activity/SplashScreen.java

```diff
@@ -80,6 +111,31 @@ public void handleEvent(String event, Object data) {
         }
     }
 
+    private boolean checkPlayServices() {
+        GoogleApiAvailability apiAvailability = GoogleApiAvailability.getInstance();
+        int resultCode = apiAvailability.isGooglePlayServicesAvailable(this);
+        if (resultCode != ConnectionResult.SUCCESS) {
+            if (apiAvailability.isUserResolvableError(resultCode)) {
+                apiAvailability.getErrorDialog(this, resultCode, PLAY_SERVICES_RESOLUTION_REQUEST)
+                        .show();
+            } else {
+                finish();
+            }
+            return false;
+        }
+        return true;
+    }
```

Comment:
checkPlayServices is duplicated a lot. Should extract this method to a util class for re-use